js/content/community.js

```diff
@@ -3546,10 +3546,14 @@ class WorkshopPageClass {
             (new StatsPageClass());
             break;
 
-        case /^\/sharedfiles\/.*/.test(path):
+        case /^\/sharedfiles\/filedetails\/?$/.test(path):
             (new WorkshopPageClass());
             break;
 
+        case /^\/sharedfiles\/editguide\/?$/.test(path):
+            Array.from(document.getElementsByName("tags[]")).forEach(tag => tag.type = "checkbox");
```

Comment:
Although it's a one-liner, this should be wrapped into a function (inside a class)

Reply:
I've seen this done elsewhere too. But you're right, I should change it. Array.from isn't neccesary here. And maybe it's better to use a for loop.

Reply:
Yup, `Array.from` is not required, but you can leave the `forEach` if you want